examples/client.c

```diff
@@ -1025,8 +1042,17 @@ get_session(
     }
     freeaddrinfo( result );
   } else {
-    if ( identity && key && (proto == COAP_PROTO_DTLS || proto == COAP_PROTO_TLS) )
-      session = coap_new_client_session_psk( ctx, NULL, dst, proto, identity, key, key_len );
+    if (cert_file && (proto == COAP_PROTO_DTLS || proto == COAP_PROTO_TLS)) {
+      coap_dtls_pki_t dtls_pki;
+      memset (&dtls_pki, 0, sizeof(dtls_pki));
+      dtls_pki.public_cert = cert_file;
+      dtls_pki.private_key = cert_file;
+      session = coap_new_client_session_pki(ctx, NULL, dst, proto, &dtls_pki);
+    }
+    else if ((identity || key) &&
```

Comment:
For better readability, I would like to have parentheses about the if-then-else cases.

Reply:
Just updated the code